### PR TITLE
Meshery UI permissions addition/fix

### PR DESCRIPTION
**Pull request by dragon-slayer875**
**Notes for Reviewers**

This PR adds missing permissions for various actions in -
- Header
- Dashboard
    - ConnectionChart
    - DashboardMeshModelGraph
    - MesheryConfigurationChart
    - WorkloadChart  
- Filters
- Designs
    - Configurator 
    - CustomToolbarSelect
    - MesheryPatterns
    - MesheryPatternCards
- Environments
    - EnvironmentsCard
- Lifecycle
    - TransferList 
- Workspaces
    - WorkspaceCard
- User menu
- Settings
- MesheryMetrics

This also fixes -
- Settings displaying a blank screen when user doesn't have permissions.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/ConfirmationModal.js
Comment (Yashsharma1911) [inline ui/components/ConfirmationModal.js:395]: We can simplify this, any reason we should stick with the change you made?

```suggestion
              disabled={!CAN(keys.UNDEPLOY_DESIGN.action, keys.UNDEPLOY_DESIGN.subject) || !disabled}
```

**Review by Yashsharma1911 — DISMISSED** [review]
Other LGTM

File: ui/components/ConfirmationModal.js
Comment (dragon-slayer875) [reply ui/components/ConfirmationModal.js:395]: @Yashsharma1911 I wrote it that way because we need to first check if the person has the permission to do that action or not, and then check if the disabled property (which is dependent on the length of the k8s context.) is true or false.
Your suggestion has the chance to enable the action even if the person doesn't have permission for that action.

Comment (dragon-slayer875) [reply ui/components/ConfirmationModal.js]: Though I just realised the '!' mark before diabled isn't needed and have updated the PR for the same.

**Review by Yashsharma1911 — APPROVED** [review]
LGTM, please keep your eye after merging this to make sure we don't introduce any error or issue

**Comment by Yashsharma1911** [comment]
Ohh ho, I missed the pr do not merge label, should we revert this PR? is this contain any breaking change?

**Pull request merged**
